framework/test-vectors/README.md

```diff
@@ -0,0 +1,75 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+## Summary
```

Comment:
We need something high level to simply explain what we're doing here. Are these "vectors" to be used with higher level ESDK/DBESDK to produce ciphertext that we can test interoperability with?

Or is this self-contained and we going to have actual "encryption materials"/"decryption materials" vectors somehow serialized to be tested across the different languages?

Reply:
Both. This starts out as only encryption/decryption materials.
But it also provides some primitives that ESDK/DBESDK can use.
Namely "how to build keyring/cmm"